book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -318,4 +323,4 @@ Conflicts:
 #
 ----
 
-You can modify that message with details about how you resolved the merge if you think it would be helpful to others looking at this merge in the future – why you did what you did, if it's not obvious.
+如果你觉得上述的信息不够充分，不能完全体现分支合并的过程，你可以修改上述信息，添加一些细节告诉未来的读者，你是如何解决合并冲突的，以及理由是什么。
```

Comment:
>what you did
你是如何解决合并冲突的

其实但看 `what` 应该翻译成 `你为解决合并冲突做了什么`，但是我认为上面那句更好。

>it would be helpful to others looking at this merge in the future
这个确实没有完全翻译出来，不如改成`对未来检视这个合并的读者提供帮助` 。